client/actions/client-track.js

```diff
@@ -0,0 +1,48 @@
+'use server'
+
+import axios from'axios';
```

Comment:
I'm not sure there's any use for axios these days? I think your code would look pretty much identical with fetch